Formula/tesseract.rb

```diff
@@ -49,15 +44,29 @@ def install
     ENV.cxx11
 
     system "./autogen.sh"
-    system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking"
+    system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking", "--datarootdir=#{HOMEBREW_PREFIX}/share"
+
+    system "make"
+
+    inreplace "tessdata/Makefile", "datarootdir = #{HOMEBREW_PREFIX}/share", "datarootdir = #{share}"
```

Comment:
Why is that necessary?

Reply:
As I commented above https://github.com/Homebrew/homebrew-core/pull/36988#issuecomment-463747370, it was not absolutely necessary to use the `inreplace` functions. The `datarootdir` must be corrected before `make install` to avoid a permission error, but the same effect could be obtained by passing  `datarootdir=#{share}` to `make install`. I used the `inreplace` functions to explicitly show which makefiles were affected by this change. If you prefer, I could remove them in favor of passing the argument as discussed above.

Reply:
Yes, please pass the argument instead, I think it would be more robust in the future (if another makefile starts using `datarootdir`). Also please add a short comment as to why we're doing this.